packages/spotlight/Spottable/Spottable.js

```diff
@@ -176,58 +174,26 @@ const Spottable = hoc(defaultConfig, (config, Wrapped) => {
 			super(props);
 			this.isFocused = false;
 			this.isHovered = false;
+
+			this.state = {
+				spottableDisabled: false
+			};
 		}
 
-		componentDidMount () {
-			// eslint-disable-next-line react/no-find-dom-node
-			this.node = ReactDOM.findDOMNode(this);
+		componentWillReceiveProps () {
+			this.setState((state, nextProps) => {
+				const spottableDisabled = this.isFocused && nextProps.disabled || nextProps.spotlightDisabled;
+				return !this.state.spottableDisabled && spottableDisabled ? {spottableDisabled} : null;
```

Comment:
Should be accessing `state` rather than `this.state` within a `setState` callback